### ISSUE TITLE
redux-form Field does not work correctly with Material-UI's SelectField when using keyboard navigation.

### ISSUE THREAD

**Issue by dcporter44** [erikras/redux-form#2764]
### Are you submitting a **bug report** or a **feature request**?
Bug Report

### What is the current behavior?
When attempting to select an option in Material-UI's SelectField with the keyboard, the field value becomes empty.

To reproduce, use tab to focus the SelectField, then attempt to select an option using the arrow keys and Return key.

You can test this on your own examples on your website:
http://redux-form.com/6.6.1/examples/material-ui/

This bug does not occur when using Material-UI by itself. Only when used with redux-form.

### What's your environment?
Using redux-form 6.5.0 with Google Chrome

**Comment by dcporter44** [comment]
I also noticed that if you tab the SelectField into focus, then tab onto the next field, the SelectField will reset to empty. You don't even have to select an option for it to reset to empty.

**Comment by jaredSiebert** [comment]
I encountered the same issue. If you take a look at erikras redux-form-material-ui plugin, you'll see how this is handled - https://github.com/erikras/redux-form-material-ui/blob/v4.1.3/src/SelectField.js

The key is lines 10 and 17.. You can add the following to get this working:
`value={input.value}`
`onBlur={() => input.onBlur(input.value)}`

Not heavily tested, but it appears to be working for me at the moment.

**Comment by dcporter44** [comment]
@jaredSiebert Ah, excellent thank you. The onBlur did the trick. Thanks!

**Comment by dcporter44** [comment]
It would be nice if the field could stay in focus afterward so that a user is able to tab to the next field. This is an feature request for Material-UI though, not redux-form.

**Comment by vancetran** [comment]
@jaredSiebert Thanks! This worked for me as well.

**Comment by gustavohenke** [comment]
~1.5 years without a comment here, so I'm closing!
